AsyncDisplayKit/ASCollectionView.h

```diff
@@ -205,17 +205,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)setTuningParameters:(ASRangeTuningParameters)tuningParameters forRangeMode:(ASLayoutRangeMode)rangeMode rangeType:(ASLayoutRangeType)rangeType ASDISPLAYNODE_DEPRECATED_MSG("Use ASCollectionNode method instead.");
 
-/**
- * Scrolls the collection to the given item.
- *
- * @param indexPath The index path of the item.
- * @param scrollPosition Where the row should end up after the scroll.
- * @param animated Whether the scroll should be animated or not.
- */
+- (nullable __kindof UICollectionViewCell *)cellForItemAtIndexPath:(NSIndexPath *)indexPath ASDISPLAYNODE_DEPRECATED_MSG("Use ASCollectionNode method instead.");
```

Comment:
It does exactly that. This was added to mirror the UITableView and UICollectionView methods. They all use `__kindof Cell *` (except one method, which looks like a mistake.)